pkg/controller/storagecluster/reconcile.go

```diff
@@ -23,15 +26,19 @@ import (
 	statusutil "github.com/openshift/ocs-operator/pkg/controller/util"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	rook "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
-	corev1 "k8s.io/api/core/v1"
 )
 
 const (
 	nodeAffinityKey   = "cluster.ocs.openshift.io/openshift-storage"
 	nodeTolerationKey = "node.ocs.openshift.io/storage"
 )
 
-var defaultMonCount int = 3
+var validTopologyLabelKeys = []string{
```

Comment:
Can we make use of keys from https://github.com/kubernetes/api/blob/master/core/v1/well_known_labels.go instead?

Reply:
Not strictly, since we'll want to provide our own labels in the future. Also I'm primarily interested in the left-hand side of the label and not the right.

Reply:
I think what umanga means is to replace the two literals here with references to the consts in that file.

Reply:
Yes, and I pointedly don't want to for the reasons listed above. :)